src/test/java/streams/part1/exercise/Exercise2.java

```diff
@@ -87,7 +85,7 @@ public void combineMapsAndPutToLatest() {
         newValues.put(alex, Status.DECLINED);
         newValues.put(helen, Status.PENDING);
 
-        // TODO реализация
+        oldValues.forEach((k, v) -> newValues.merge(k, v, (oldValue, newValue) -> oldValue));
```

Comment:
Seems like the task was to use "latest" enum, according to their provided sorting. It means using enum.ordinal()

Reply:
The word latest(should be latter actually, like "former and latter") refers to the latter map i.e. newValues, therefore the solution is correct